core/common/src/main/java/alluxio/wire/InconsistentProperty.java

```diff
@@ -104,10 +105,25 @@ public int hashCode() {
   public String toString() {
     return Objects.toStringHelper(this)
         .add("key", mName)
-        .add("values", mValues)
+        .add("values", getValuesWithoutOptionalKeyword())
```

Comment:
the more general purpose of this method is to format the values to make them easier to read, could we name this `formatValues` instead? Also good to make it static and take `mValues` as an argument to preserve the symmetry of `.add("varName", var)`